### PR TITLE
Add IXO to assetlists

### PR DESCRIPTION
**Pull request by jayjay-crypto**

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -374,5 +374,33 @@
...
+      "denom_units": [
+        {
+          "denom": "ibc/F3FF7A84A73B62921538642F9797C423D2B4C4ACB3C7FCFFCE7F12AA69909C4B",
+          "exponent": 0,
+          "aliases": [
+            "uixo"
+          ]
+        },
+        {
+          "denom": "IXO",
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:388]: I'm going to recommend making this denom be lowercase, as that seems to be the standard for some reason

Comment (sunnya97) [reply osmosis-1/osmosis-1.assetlist.json:388]:
```suggestion
          "denom": "ixo",
```

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -374,5 +374,33 @@
...
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/F3FF7A84A73B62921538642F9797C423D2B4C4ACB3C7FCFFCE7F12AA69909C4B",
+      "name": "IXO",
+      "display": "IXO",
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:394]:
```suggestion
      "display": "ixo",
```

**Review by sunnya97 — CHANGES_REQUESTED** [review]

**Review by sunnya97 — APPROVED** [review]

File: osmosis-1/osmosis-1.assetlist.json [hunk 3]
```diff
@@ -374,5 +374,33 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/juno.svg"
       }
     }
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:376]:
```suggestion
    },
```

File: osmosis-1/osmosis-1.assetlist.json [hunk 4]
```diff
@@ -373,6 +373,34 @@
...
+      "base": "ibc/F3FF7A84A73B62921538642F9797C423D2B4C4ACB3C7FCFFCE7F12AA69909C4B",
+      "name": "IXO",
+      "display": "ixo",
+      "symbol": "IXO",
+      "ibc": {
+        "source_channel": "channel-4",
+        "dst_channel": "channel-38"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/ixo.png",
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:401]:
```suggestion
        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/ixo.png"
```

**Comment by jayjay-crypto** [comment]
Thanks for the review

**Pull request merged**